boards/nxp/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -100,49 +126,17 @@
 			   <20 0 &gpio1 17 0>,	/* D14 */
 			   <21 0 &gpio1 16 0>;	/* D15 */
 	};
-
-	panel {
-		compatible = "rocktech,rk043fn02h-ct";
-		port {
-			lcd_panel_in: endpoint {
-				remote-endpoint = <&lcd_panel_out>;
-			};
-		};
-	};
 };
 
-arduino_i2c: &lpi2c1 {};
+nxp_touch_i2c: &lpi2c1 {};
 
-&lcdif {
-	status = "okay";
-	width = <480>;
-	height = <272>;
-	display-timings {
-		compatible = "zephyr,panel-timing";
-		hsync-len = <41>;
-		hfront-porch = <4>;
-		hback-porch = <8>;
-		vsync-len = <10>;
-		vfront-porch = <4>;
-		vback-porch = <2>;
-		de-active= <1>;
-		pixelclk-active = <1>;
-		hsync-active = <0>;
-		vsync-active = <0>;
-		clock-frequency = <9210240>;
-	};
-	pixel-format = <PANEL_PIXEL_FORMAT_BGR_565>;
-	data-bus-width = "16-bit";
+zephyr_lcdif: &lcdif {
```

Comment:
Should it be `nxp_lcdif` as lcdif is NXP specific ?

Reply:
Our IP is named LCDIF, but the `zephyr_lcdif` is not NXP specific- the properties we use to describe the display panel in the shield are generic across different LCD controllers. For more info, take a look at the discussion here: https://github.com/zephyrproject-rtos/zephyr/pull/55493#discussion_r1194019244, or the PR where the panel timings were standardized: https://github.com/zephyrproject-rtos/zephyr/pull/53907

Reply:
Ok, thanks. BTW, as display is now removed in rt1064 dts, I will update [my PR](https://github.com/zephyrproject-rtos/zephyr/pull/72633) to use the display instead.